framework/src/source/BaseTestSuite.bs

```diff
@@ -15,18 +15,18 @@ namespace rooibos
     protected hasIgnoredTests = false
     protected isSolo = false
     protected isIgnored = false
-    protected noCatch = false
+    public noCatch = false
```

Comment:
Many changes in the .BS files to support proper typing and to fix validation errors